docs/configuration/config-file-reference.md

```diff
@@ -1774,10 +1774,15 @@ bigtable:
     # CLI flag: -bigtable.grpc-max-send-msg-size
     [max_send_msg_size: <int> | default = 16777216]
 
-    # Use compression when sending messages.
+    # Deprecated: Use gzip compression when sending messages.
     # CLI flag: -bigtable.grpc-use-gzip-compression
     [use_gzip_compression: <boolean> | default = false]
 
+    # Use compression when sending messages. Supported values are: 'gzip',
+    # 'snappy' and '' (disable compression)
+    # CLI flag: -bigtable.grpc-compression
+    [compression: <string> | default = ""]
```

Comment:
I would expect this to be `grpc_compression`. I'm not sure why the CI linter passed. Could you run `make doc` again, please?

Reply:
as you may know, CI don't run.

Reply:
Oh, I haven't noticed! Can't understand why the CI is not running on this specific PR 👀